src/crypto_utils.ts

```diff
@@ -35,6 +34,17 @@ export function urlSafe(buffer: Uint8Array): string {
   return encoded.replace(/\+/g, '-').replace(/\//g, '_').replace(/=/g, '');
 }
 
+// adapted from source: http://stackoverflow.com/a/11058858
+export function str2ab(str: string) {
```

Comment:
Lets call this method something like `textEncodeLite()`. 
Also you should add a comment that describes why we did this in the first place with a caniuse link.

Reply:
Will do - thanks!